recipe/meta.yaml

```diff
@@ -13,10 +15,12 @@ source:
   sha256: fad60d606c02e6164018692c6c9f2c159a9130c2bf32e8c5f4f1b6ba2dda2b68
 
 build:
-  number: 0
+  number: {{ build }}
   string: >-
-    nompi_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [mpi == 'nompi']
-    mpi_{{ mpi }}_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [mpi != 'nompi']
+    nompi_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [mpi == 'nompi' and double == 'no']
+    mpi_{{ mpi }}_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [mpi != 'nompi' and double == 'no']
+    nompi_dblpres_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [mpi == 'nompi' and double == 'yes']
+    mpi_{{ mpi }}_dblpres_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [mpi != 'nompi' and double == 'yes']
```

Comment:
Just noticed I put `dblpres` instead of `dblprec`. Not a big error, but that spelling mistake is going to bug me. Could we change this to `dblprec`, or `dbl`?

Reply:
Sure thing - I have a truly marginal preference for `dblprec` of those two. If anybody feels stronger than zero I'm happy to change it around again